### PR TITLE
🎉;feat:home page overview

### PR DESCRIPTION
**Pull request by Max-Mahinda**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshot (optional)
Screenshot from 2023-09-28 15-57-46

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -3,3 +3,94 @@
...
+  margin-top: 15%;
+  max-width: 60%;
+  
+  margin-right: 20%;
+
+   
+   
+  
+
+
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css:29]: unnecessary space

File: src/css/style.css [hunk 2]
```diff
@@ -3,3 +3,94 @@
...
+}
+
+
+body{
+  background-color: #DBC2F5;
+}
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css:35]: declare parents elements at the top

File: src/css/style.css [hunk 3]
```diff
@@ -3,3 +3,94 @@
...
+
+.pic img {
+  padding-right: 20%;
+  margin-top: 15%; 
+  margin-left: 20%;
+  margin-bottom: 20%;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css:41]: use a single margin property i.e margin:10 10 10 10;

File: src/css/style.css [hunk 4]
```diff
@@ -3,3 +3,94 @@
...
+ 
+/* For Mobile Phones Portrait or Landscape View */
+@media screen
+    and (max-device-width: 640px) {
+    .gfg-div {
+        width: 400px;
+        height: 200px;
+        background-color: chartreuse;
+        color: black;
+    }
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css:95]: there is repetition here, be keen

File: src/css/style.css [hunk 5]
```diff
@@ -3,3 +3,94 @@
...
+/* For Mobile Portrait View */
+@media screen and (max-device-width: 480px)
+    and (orientation: portrait) {
+    .gfg-div {
+        width: 200px;
+        height: 200px;
+        background-color: red;
+        color: #fff;
+    }
+}
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css:74]: since this is the smallest device width, i recommend it goes to the bottom of other larger device widths

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
please go through your files again and optimize it

**Comment by Max-Mahinda** [comment]
Done

**Comment by Max-Mahinda** [comment]
Dear sir,
Submitted few projects but haven’t gotten any response yet from you guys.

Regards,
Max Mwangi.

**Comment by Max-Mahinda** [comment]
Heey i have also submitted so many but cant see them in the merging list? I
dont understand why

**Comment by JAPHETHNYARANGA** [comment]
Good attempt but this issue has been closed . Please pick another open issue. Hint; don't delete code already in a file when working on it.

**Pull request closed without merging**